lib/assets/big_query/main.js

```diff
@@ -0,0 +1,94 @@
+export function init(ctx, info) {
+  ctx.importCSS("main.css");
```

Comment:
Can I import the `main.css` from DbConnection? Since it's a duplicated file from other SmartCell

Reply:
Assets for each `Kino.JS` must be in a separate directory, however we could have `shared.css` and create symlink to it in each directory.

We plan to split kino into several packages before the release, so let's keep it simple for now, we can revisit CSS to reduce duplication later. Also, if we manage to merge this cell with Db connection, we won't have the problem altogether :)

Reply:
Nice